doc/quick_start/contributor-guide.rst

```diff
@@ -0,0 +1,289 @@
+when patches are needed against different components. 
+
+Patches may not be submitted against this aggregated repository; send those to the original layer and
+don't mix changes against different components in one PR.
+
+To begin then, you’ll need to identify which layer needs to be modified.
+
+In this example, we assume you’re familiar with Git, GitHub, and the basic Linux development process. For convenience,
+we'll use both the http://github.com/ostroproject web interface and Git command line tools.
+
+Upstream project code is not stored in the Ostro Project's Git tree. Instead, 
+there are Yocto Project recipes that reference upstream source and, as needed, hotfix patches for 
+issues that have not been incorporated upstream. 
+
+We’ll assume in the following steps that you’ll be modifying the ``meta-ostro`` layer.
+
+If you haven't already done so, you'll need to create a (free) GitHub account on http://github.com and have
+Git tools available on your development system.  (For Windows users you can use `Git Bash`_ or other Git command line tools.)
+
+.. _`Git Bash`: https://git-for-windows.github.io/
+
+Prepare your patch
+------------------
+
+.. _`Fork a Repo`: https://help.github.com/articles/fork-a-repo/
+.. _`Yocto Project Managing Layers`: http://www.yoctoproject.org/docs/current/dev-manual/dev-manual.html#managing-layers
+
+#. Create a Fork (using GitHub's web interface)
+   In GitHub, create a fork of the repo containing the layer you need to modify. In this example we use ``meta-ostro``. 
+   For more information, see GitHub's `Fork a Repo`_ help page.
+
+   In your web browser, navigate to the repo: https://github.com/ostroproject/meta-ostro and click on the Fork button
+   in the top right corner to fork your own copy of the ``ostroproject/meta-ostro`` repo to your account.
+
+#. Create a repository on your local computer to your fork.  If you have ssh keys generated you can register your 
+   public key on your GitHub account (SSH and GPG keys in your Personal Settings on Git Hub) to be authorized.  
+   Otherwise, you can clone with "https" and specify your GitHub username and password:::
+ 
+      $ git clone github.com:<your-username>/meta-ostro                # if you've registered your ssh key
+      $ git clone https://github.com/<your-username>/meta-ostro.git    # if not use this (and your GitHub username/password)
+
+      $ cd meta-ostro
+
+      $ git remote add upstream github.com:ostroproject/meta-ostro            # if you've registered your ssh key
+      $ git remote add upstream https://github.com/ostroproject/meta-ostro    # if you cloned with https
+
+      $ git remote -v        # verify origin (your fork) and remote (Ostro OS master) are defined as expected
+
+#. Create a new branch to work on your patch:::
+
+      $ git fetch upstream
+      $ git checkout -b my-patched-branch upstream/master
+
+#. Make and test your changes
```

Comment:
I'm wondering how obvious it is to the user at this point he needs to go back to ostro-os